crates/criticaltrust/src/revocation_info.rs

```diff
@@ -5,29 +5,42 @@ use crate::keys::KeyRole;
 use crate::signatures::Signable;
 use crate::NoRevocationsCheck;
 use serde::{Deserialize, Serialize};
-use time::macros::datetime;
 use time::OffsetDateTime;
 
 /// Holds hashes of revoked content which are included as a part of the [`KeysManifest`].
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct RevocationInfo {
-    pub revoked_content_sha256: Vec<String>,
+    pub revoked_content_sha256: Vec<RevocationContentSha256>,
     #[serde(with = "time::serde::rfc3339")]
     pub expires_at: OffsetDateTime,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+pub struct RevocationContentSha256 {
```

Comment:
This looks like a newtype and may be better suited as `pub struct RevocationContentSha256(#[serde(with = "crate::serde_base64")] Vec<u8>)`

Reply:
Ah! Yes, thank you so much!